payload/game/system/RaceConfig.cc

```diff
@@ -128,6 +128,19 @@ void RaceConfig::applyCPUMode() {
     }
 }
 
+void RaceConfig::initRace() {
+    REPLACED(initRace)();
+    m_spRace = m_spMenu;
+}
+void RaceConfig::initAwards() {
+    REPLACED(initAwards)();
+    m_spRace = m_spMenu;
+}
+void RaceConfig::initCredits() {
+    REPLACED(initCredits)();
+    m_spRace = m_spMenu;
+}
+
```

Comment:
```suggestion
void RaceConfig::initRace() {
    REPLACED(initRace)();
    m_spRace = m_spMenu;
}

void RaceConfig::initAwards() {
    REPLACED(initAwards)();
    m_spRace = m_spMenu;
    m_spAwards = m_spRace;
}

void RaceConfig::initCredits() {
    REPLACED(initCredits)();
    m_spMenu = m_spAwards;
    m_spRace = m_spMenu;
}
```

This more accurately reflects the base game, so we should likely mirror it with our extended scenarios.

Reply:
Sure, done